api/buffer/design.go

```diff
@@ -298,6 +298,19 @@ var _ = Service("buffer", func() {
 			PayloadTooLargeError()
 		})
 	})
+
+	Method("GetTask", func() {
+		Meta("openapi:summary", "Get task")
+		Description("Get details of a task.")
+		Result(Task)
+		Payload(GetTaskRequest)
+		HTTP(func() {
+			GET("/receivers/{receiverId}/tasks/{type}/{createdAt}/{taskId}")
```

Comment:
@JakubMatejka  
Ja by som dal `{createdAt}/{taskId}` iba ako `{taskId}`, ze je sucastou ID aj datum nemusi uzivatela trapit.

Ked to nechces parsovat tak uprav `TaskKey`, aby tam neboli 2 fields `date + random suffix`, ... ale rovno iba TaskID ako string.

A teda nebudes musiet nic parsovat.

Testy a etcd zostanu bez zmeny a na tom jednom mieste, kde sa generuje TaskKey to upravis.

Co povies?

Reply:
Resp. v etcd bude drobna zmena, ze v JSON objekte nebudu tiez 2 fields (date, random suffix), ale iba jeden - TaskID.

Reply:
CreatedAt moze byt samostatny field.

Mozem poslat PR ak ti to pomoze....

Reply:
@JakubMatejka tak som to pripravil, este bezia testy https://github.com/keboola/keboola-as-code/pull/1191#pullrequestreview-1293459199

Reply:
Aha :slightly_smiling_face: https://github.com/keboola/keboola-as-code/pull/1192

Si neodpisal, tak som myslel, ze riesis nieco ine, tak som ti to chcel urychlit, ... aby si zase nemusel implmentovat, co sim si ja vymyslel.

Tak si mozes vybrat,  urcite pozri komentare v mojom PR.

Reply:
To jsi hodnej, zrovna jsem ti chtěl psát že mi tam zase padají oblíbený testy a to jsem kopíroval a diffoval ve VS Code a pořád nic nevidím. 😅 Tak se aspoň podívám k tobě, případně to rovnou použiju. Díky! Podívám se o víkendu, šel jsem už raději bourat zeď na dvířka do podkrovní komory. To je krásně jednoduchá věc. 😄 

<img width="1089" alt="image" src="https://user-images.githubusercontent.com/215660/218149899-e58455e1-9ec9-47d8-9422-633f6ab92291.png">

Reply:
No vida, trocha bourání a už tam ten bug vidím. 😃 
![CleanShot 2023-02-10 at 17 57 54](https://user-images.githubusercontent.com/215660/218150863-68112927-ee17-44af-bec9-f63498901123.png)